recipes/bezier/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "bezier" %}
+{% set version = "0.2.1" %}
```

Comment:
The latest release appears to be `0.3.0`. Is there a reason not to build the recipe for the latest release?